codecov.yml

```diff
@@ -19,3 +19,6 @@ coverage:
 comment:
   layout: "header, diff"
   behavior: default
+
+ignore:
+  - package-lock.json
```

Comment:
mmm I'm using Visual Code...I am not sure if I need to do something on my side 😄